-inputs.tf

```diff
@@ -0,0 +1,34 @@
+variable "input_tags" {
+  description = "Map of tags to apply to resources"
+  type        = map
+  default = {
+    Developer   = "StratusGrid"
+    Provisioner = "Terraform"
+  }
+}
+
+variable "name" {
```

Comment:
usually use something like var.name_prefix and var.name_suffix and interpolate a resource's name using both of those variables.

Reply:
added.